### PR TITLE
Milestone 4 :  Refactor with hooks.

### PR DESCRIPTION
**Pull request by Chrissiku**
### What I use 🧾   
- I Use React hooks. 
- I  Use functional components.

## Technology used 🧑‍💻 
- Language : JavaScript, JSX and CSS3
- Library: `React.js` 
- Module Bundler: `Webpack`
- Compiler: `Babel`
- Best Practice : Github flow, Gitflow, Git and Linters

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,71 +1,58 @@
...
 import SpanTwo from './component_mocks/spanTwoButton';
 import Display from './component_mocks/Display';
 import OperationButton from './component_mocks/OperationButton';
 import calculate from '../logic/calculate';
 
-class App extends React.PureComponent {
-  constructor(props) {
-    super(props);
+function App() {
+  const [data, setData] = useState({
```
Comment (Wes-Isaac) [inline src/components/Calculator.js]: - [x] [_OPTIONAL_] It's best practice to name functions the same as the file name in this case` function Calculator`.

**Review by Wes-Isaac — APPROVED** [review]
Hi @Chrissiku, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- README looks professional👀.
- Project works as expected 🚀.
- Linters are all green ✔️.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Chrissiku) [reply src/components/Calculator.js]: Sure, 
Thank you @Wes-Isaac, Let me rename before I merge

**Comment by Chrissiku** [comment]
Thank you, @Wes-Isaac  for this project approval

**Pull request merged**